virtcontainers/factory/factory.go

```diff
@@ -140,7 +139,7 @@ func checkVMConfig(config1, config2 vc.VMConfig) error {
 func (f *factory) checkConfig(config vc.VMConfig) error {
 	baseConfig := f.base.Config()
 
-	return checkVMConfig(config, baseConfig)
+	return checkVMConfig(baseConfig, config)
```

Comment:
This change seems to have no effect so do we really need to do this?

Reply:
It's a clean up that the log otherwise what the log says new is actually old base config right now.

Reply:
ack - thanks @bergwolf.